Ryujinx.Ava/UI/Models/InMemoryLogTarget.cs

```diff
@@ -0,0 +1,78 @@
+using Avalonia.Media;
+using Ryujinx.Common.Logging;
+using System.Collections.ObjectModel;
+
+namespace Ryujinx.Ava.UI.Models
+{
+    public class InMemoryLogTarget : ILogTarget
+    {
+        public class Entry
+        {
+            public Color Color { get; set; }
+            public string Text { get; set; }
+        }
+
+        private static Color GetLogColor(LogLevel level) => level switch {
+            LogLevel.Info    => Colors.White,
+            LogLevel.Warning => Colors.Yellow,
+            LogLevel.Error   => Colors.Red,
+            LogLevel.Stub    => Colors.DarkGray,
+            LogLevel.Notice  => Colors.Cyan,
+            LogLevel.Trace   => Colors.DarkCyan,
```

Comment:
```suggestion
            LogLevel.Warning => Color.FromRgb(235, 205, 85),
            LogLevel.Error   => Color.FromRgb(255, 95, 85),
            LogLevel.Stub    => Colors.DarkGray,
            LogLevel.Notice  => Color.FromRgb(2, 197, 229),
            LogLevel.Trace   => Color.FromRgb(200, 146, 245),
```

Just a little cosmetic suggestion, you can ignore it if you don't like.

![image](https://user-images.githubusercontent.com/4905390/211211583-a8516bae-87e3-452c-9948-d48e79126f1d.png)

Reply:
Personally I prefer the current colors, they are slightly brighter which makes it easier to read on a dark background.

Reply:
It's just following some of the Switch colors scheme

Reply:
Suggested colors look great!
During testing I didn't notice any readability issues, but if they make messages harder to read, we should probably adjust them a bit.